apps/client/src/app/components/user-account-settings/user-account-settings.scss

```diff
@@ -6,6 +6,13 @@
     font-size: 90%;
     line-height: 1.2;
   }
+
+  ::ng-deep .mat-mdc-slide-toggle {
```

Comment:
Strange! Do you have an idea why this is needed and not working out of the box?

Reply:
For some reason `mat-slide-toggle` is not honouring the color theme though `color=primary` defined in HTML.

It could be due to https://github.com/angular/components/issues/26818.

Reply:
Thanks for your research! 👍🏻